src/ResourceManagement/Insights/Microsoft.Azure.Insights/Generated/Management/Insights/Models/Recurrence.cs

```diff
@@ -27,11 +27,11 @@ public partial class Recurrence
         /// <param name="frequency">the recurrence frequency. How often the
         /// schedule profile should take effect. This value must be Week,
         /// meaning each week will have the same set of profiles. Possible
-        /// values include: 'None', 'Second', 'Minute', 'Hour', 'Day',
-        /// 'Week', 'Month', 'Year'</param>
+        /// values include: 'None', 'Second', 'Minute', 'Hour', 'Day', 'Week',
+        /// 'Month', 'Year'</param>
         /// <param name="schedule">the scheduling constraints for when the
         /// profile begins.</param>
-        public Recurrence(RecurrenceFrequency? frequency = default(RecurrenceFrequency?), RecurrentSchedule schedule = default(RecurrentSchedule))
+        public Recurrence(RecurrenceFrequency frequency, RecurrentSchedule schedule)
```

Comment:
Fixing a bug in the spec.
The RecurrenceFrequency is already mandatory in PowerShell when the autoscale profile is using a recurrence (see New-AzureRmAutoscaleProfile cmdlet). Same is true for ScheduledDays, ScheduleHours, ScheduleMinutes, ScheduleTimeZone which are needed to create the RecurrentSchedule. So I won't break PowerShell or Node.js with this change, I am only stating a fact.